projects/robots/mir/mir100/worlds/mir100.wbt

```diff
@@ -288,30 +278,30 @@ Table {
 }
 WoodenChair {
   translation 2.921084 1.648051 0
-  rotation 0.092691584028185 -0.704062593186979 -0.704062593186979 2.956737685995372
+  rotation 7.19236e-09 -5.51889e-09 1 1.83259
```

Comment:
```suggestion
  rotation 0 0 1 1.83259
```

Reply:
Personally I don't think it brings much to the table to keep these ultra precise axis of rotation in world files for objects in a scene. I mark the suggestion, but it's up to you to decide if accept them or not. I'm fine either way.

Reply:
I agree with you